components/pingfederate.key.manager/src/main/java/org/wso2/pingfederate/client/PingFederatKeyManagerClient.java

```diff
@@ -199,7 +199,12 @@ public OAuthApplicationInfo updateApplication(OAuthAppRequest oAuthAppRequest) t
 
         if (oAuthAppRequest.getOAuthApplicationInfo() != null) {
             ClientInfo clientInfo = fromOauthAppRequestToClientInfo(oAuthAppRequest);
-            pingFederateDCRClient.updateApplication(toClientInfoList(clientInfo));
+            Response response = pingFederateDCRClient.updateApplication(toClientInfoList(clientInfo));
+            if (response.status() == HttpStatus.SC_OK) {
+                return fromClientInfoToOauthApplicationInfo(clientInfo);
+            } else {
+                throw new APIManagementException("Error while updating application" + response.status());
```

Comment:
This will print the status code just after the word application. Shall we format this exception message to a format like:
throw new APIManagementException("Error while updating application. Response status code: " + response.status());